src/Look/HorizonLook.cpp

```diff
@@ -7,11 +7,17 @@
 void
 HorizonLook::Initialise()
 {
-  aircraft_pen.Create(Layout::Scale(2), COLOR_BLACK);
+  aircraft_pen.Create(Layout::Scale(1), Color(0xf2, 0xf2, 0x19));
+  aircraft_brush.Create(Color(0xf2, 0xf2, 0x19));
```

Comment:
Plsease make a definition for Color(0xf2, 0xf2, 0x19)  in HorizionLook.hpp, and reuse the definition here.